### ISSUE TITLE
Unify source and executable names

### ISSUE THREAD

**Issue by jakobj** [incf-music/music-adapters#4]
I find it very confusing that the binaries for the adapters are all_lower_case_with_underscores while the source files are CamelCase. Is it just me or would it make sense to have a common convention for source and binary files?

**Comment by weidel-p** [comment]
I agree. Actually I don't like that the binaries are using underscores, but I also never saw a binary written in camelcase. Maybe the nicest solution would be to simply remove all the underscores from the binary names.

**Comment by jakobj** [comment]
how about renaming sources files to underscore_convention?
